packages/snfoundry/scripts-ts/deploy.ts

```diff
@@ -42,28 +42,24 @@ import { green } from "./helpers/colorize-log";
  * @returns {Promise<void>}
  */
 const deployScript = async (): Promise<void> => {
-  const { address: marquis_core_address } = await deployContract({
```

Comment:
Since scaffold-starknet is a "template", then we have to update it to our project logic, we dont have "YourContract.cairo" , we have `MarquisCore` and `Ludo`

Reply:
I've also updated this deploy script